APIs/AuthorizationAPI.raml

```diff
@@ -4,8 +4,8 @@
 # (c) AMWA 2020
 
 title: NMOS Authorization
-baseUri: http://api.example.com/{issuer}
-issuer: x-nmos/auth/v1.0
+baseUri: https://api.example.com/{version}
```

Comment:
Light relief... I've always written `api.example.com`, but recently noticed that most of the NMOS specs have `example.api.com` and discovered that `api.com` is owned by General Electric. 😄 I think it would be better if they all used `api.example.com`...